frontend/Makefile

```diff
@@ -29,33 +29,80 @@ BUILD_DIR ?= build
 help:
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' Makefile | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
+.PHONY: help
+
+##############################################################################
+# Local Development
+##############################################################################
+
+local: | local-build local-run ## Runs the steps for local (non-docker) development
+
```

Comment:
The Makefile within the `frontend/` folder has two main targets:

1. To rebuild and run via docker containers
```
make docker
```

2. To build and run the frontend on your local (non-docker) environment
```
make local
```
Another useful local target is `make test` which will run the formatter, the linter and the unit tests on your local file-system